google/cloud/spanner/internal/defaults.cc

```diff
@@ -105,6 +105,20 @@ Options DefaultOptions(Options opts) {
       (std::min)(min_sessions, max_sessions_per_channel * num_channels);
 
   if (!opts.has<spanner::RouteToLeaderOption>()) {
+#if 1
```

Comment:
Do we want to leave the `'#if` here or just let version control store the original version?

Reply:
I thought this was a little easier/clearer, but I'm happy to go the other way.  Done.